### PR TITLE
ENG-1503 Changing behavior when version is not found, returning now a…

### PR DESCRIPTION
**Pull request by BrenoQVDS**
…n empty array

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **brenoqvds**

If they are not already assigned, you can assign the PR to them by writing `/assign @brenoqvds` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-plugin-jpversioning).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/entando-plugin-jpversioning/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by BrenoQVDS** [comment]
The code coverage on new code is lower than the quality gate because I changed two lines that were Throwable to Exception, following the Backend Code Refactory notes. The rest of the code its tested.

**Review by alepintus — APPROVED** [review]

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/entando-plugin-jpversioning/pull/9#pullrequestreview-534266588" title="Approved">alepintus</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-plugin-jpversioning).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/entando/entando-plugin-jpversioning/blob/master/OWNERS)~~ [alepintus]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
